<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="vikram" failures="1" tests="5" name="Personal Loan" time="0.027" errors="0" timestamp="2024-12-08T20:35:42 IST">
  <testcase classname="tests.Day1" name="demo" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.Day1.demo(Day1.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="tests.Day2" name="pLoan" time="0.001"/>
  <testcase classname="tests.Day4" name="loginAPIHomeLoan" time="0.001"/>
  <testcase classname="tests.Day4" name="mobileLoginHomeLoan" time="0.0"/>
  <testcase classname="tests.Day4" name="webLoginHomeLoan" time="0.002"/>
  <testcase classname="tests.Day1" name="secondTest" time="0.0">
    <ignored/>
  </testcase> <!-- secondTest -->
</testsuite> <!-- Personal Loan -->
